tests/python/unittest/test_autotvm_config_library.py

```diff
@@ -0,0 +1,252 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""Test Config Library"""
+
+import copy
+import datetime
+import filecmp
+import json
+import os
+import random
+import tempfile
+
+from tvm.autotvm.config_library import ConfigLibrary
+from tvm.autotvm.task.dispatcher import ApplyHistoryBest
+from tvm.autotvm.tuner.callback import log_to_file
+from tvm.autotvm.record import encode
+
+from test_autotvm_common import get_sample_records
+
+
+def get_random_configs(n):
+    records = get_sample_records(n)
+    random_configs = []
+    r = random.Random()
+    r.seed(42)
```

Comment:
You can hard code random configs, but do not use random generator in the unit tests (fixing the seed number doesn't guarantee the result is deterministic on different machines) sicne it is hard to catch the problem by all means.

Reply:
Understood, I'll change this!